pyscf/lib/dft/libxc_itrf2.c

```diff
@@ -1084,27 +1100,15 @@ void LIBXC_xc_func_set_params(const int nfn, xc_func_type *fn_obj, const double
                 // set the range-separated parameter
                 if (omega[i] != 0) {
                         // skip if func is not a RSH functional
-#if XC_MAJOR_VERSION <= 7
-                        if (func->cam_omega != 0) {
-                                func->cam_omega = omega[i];
-                        }
-#else
-                        if (func->hyb_omega != NULL && func->hyb_omega[0] != 0) {
-                                func->hyb_omega[0] = omega[i];
+                        if ( xc_func_find_ext_params_name(&func, "_omega") >= 0 ) {
+                                xc_func_set_ext_params_name(&func, "_omega", omega[i]);
```

Comment:
Should be `func` instead of `*func`, since it's `*func` in L1094 while in pyscf-core it's `xc_func_type func;`. Actually there's a compiler warning when building your code.